pincer/objects/guild/guild.py

```diff
@@ -449,14 +450,14 @@ async def get_member(self, _id: int) -> GuildMember:
 
     @overload
     async def modify_member(
-        self,
-        *,
-        _id: int,
-        nick: Optional[str] = None,
-        roles: Optional[List[Snowflake]] = None,
-        mute: Optional[bool] = None,
-        deaf: Optional[bool] = None,
-        channel_id: Optional[Snowflake] = None,
+            self,
```

Comment:
Since we are using black, there should be a single indent on multi-line functions args.